src/controllers/user.js

```diff
@@ -54,5 +55,40 @@ class UserController {
       res.status(400).json({ error: error.message });
     }
   };
+
+  static getAllUsers = async (req, res) => {
+    try {
+      const users = await findAllUsers();
+      res.status(201).json(users);
+    } catch (error) {
+      res.status(400).json({ message: error.message });
+    }
+  };
+
+  static assignRoles = async (req, res) => {
+    const userRole = req.body.RoleId;
+    const userId = req.params.id;
+    const userExist = await User.findOne({
```

Comment:
This is supposed to be done in the services and you call the service here

Reply:
You can use the one in the roles service already,  "findRolesById", then add a second argument to the functions for other models to include